frontend/packages/pipelines-plugin/src/utils/pipeline-utils.ts

```diff
@@ -313,28 +314,11 @@ export const getPipelineRunWorkspaces = (
   );
 };
 
-export const calculateRelativeTime = (startTime: string, completionTime?: string) => {
+export const calculateDuration = (startTime: string, completionTime: string, list?: boolean) => {
```

Comment:
To calculate a duration without an end date it's maybe easier to switch to Date here and then parse the strings in calling functions? But that's not required.

```suggestion
export const calculateDuration = (startTime: Date, endTime: Date, long: boolean) => {
```